codalab/worker/worker.py

```diff
@@ -109,6 +110,11 @@ def __init__(
         self.pass_down_termination = pass_down_termination
         self.exit_on_exception = exit_on_exception
 
+        # Initialize sentry logging (this is a null-op if
+        # the CODALAB_SENTRY_INGEST_URL env var is not set).
+        initialize_sentry()
```

Comment:
Why do you need this both here and in `main.py`?

Reply:
Do people ever instantiate workers without using `main.py`? Was trying to cover that case

Reply:
(and i wanted it in `main.py` in case there were any exceptions between when args are parsed and when the `Worker` is constructed)

Reply:
I don't think people should be instantiating workers directly?  But I think that `initialize_sentry()` probably should just be called as early as possible in the appropriate top-level main script (e.g., rest server).